### PR TITLE
Register new package CSVFiles.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [davidanthoff/CSVFiles.jl](https://github.com/davidanthoff/CSVFiles.jl)
Release: [v0.0.1](https://github.com/davidanthoff/CSVFiles.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/davidanthoff/CSVFiles.jl/builds/241065280)
cc: @davidanthoff

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@davidanthoff This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by davidanthoff** [comment]
The JuliaCIBot builds here will fail until a new version of FileIO is tagged. But that can only happen once this package here is tagged, there is a chicken and egg situation. It seems better to first tag this package here and then FileIO, given that there are currently users of FileIO that might be affected, but no users of this package.

File: CSVFiles/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.6-
```
Comment (tkelman) [inline CSVFiles/versions/0.0.1/requires]: same comment as https://github.com/JuliaLang/METADATA.jl/pull/9733/files#r121231480

**Comment by davidanthoff** [comment]
@tkelman I believe I addressed all the specific issues you raised here.

**Comment by davidanthoff** [comment]
Could this please be merged? It was open more than 24 hours and I addressed all feedback.

**Comment by ViralBShah** [comment]
Are we waiting on JuliaCIBot?

**Comment by ViralBShah** [comment]
Merging for now.

File: CSVFiles/versions/0.0.1/requires [hunk 2]
```diff
@@ -0,0 +1,6 @@
+julia 0.6-pre
+TextParse 0.1.6
```
Comment (tkelman) [inline CSVFiles/versions/0.0.1/requires]: why does this need the absolute latest TextParse? lower bounds should be the earliest version that would work

Comment (ViralBShah) [inline CSVFiles/versions/0.0.1/requires]: TextParse is fast moving. Best to not use older versions.

Comment (tkelman) [inline CSVFiles/versions/0.0.1/requires]: setting bounds too tight causes pkg resolver errors and inability to install versions that would work together

Comment (davidanthoff) [inline CSVFiles/versions/0.0.1/requires]: I don't understand how package authors are realistically supposed to figure out a lower bound. TextParse doesn't have a NEWS, so all I can look at are the releases. v0.1.6 says "use _pointer in place of pointer", which suggests to me that previous versions had an error. But I have no clue, and I certainly don't have the time to test my package with every old version of every package I depend on to figure out what the lowest possible bound is. If there was some tool that did this automatically, great, but asking package devs to do this manually seems like an enormous misallocations of their time, quite frankly, and I want to use the time I have for julia to improve my packages.

Comment (ViralBShah) [inline CSVFiles/versions/0.0.1/requires]: I know, but all these are new packages and with quick development. Best to let things move fast and in a month or two, the bounds will get older and things will stabilize. Technically you are right, but it seems that in early stages, rules could perhaps be a bit more lax.

Comment (tkelman) [inline CSVFiles/versions/0.0.1/requires]: a few minutes looking at blame and /compare can usually get much closer to accurate than forcing latest

**Pull request merged**